src/invidious/comments.cr

```diff
@@ -587,18 +587,18 @@ def content_to_comment_html(content)
           end
         end
 
-        text = %(<a href="#{url}">#{text}</a>)
+        text = %(<a href="#{url}">#{reduce_uri(url)}</a>)
       elsif watch_endpoint = run["navigationEndpoint"]["watchEndpoint"]?
         length_seconds = watch_endpoint["startTimeSeconds"]?
         video_id = watch_endpoint["videoId"].as_s
 
         if length_seconds && length_seconds.as_i > 0
           text = %(<a href="javascript:void(0)" data-onclick="jump_to_time" data-jump-time="#{length_seconds}">#{text}</a>)
         else
-          text = %(<a href="/watch?v=#{video_id}">#{text}</a>)
+          text = %(<a href="/watch?v=#{video_id}">#{reduce_uri("/watch?v=#{video_id}")}</a>)
```

Comment:
In this case, you probably want to recreate the original youtube URL (`www.youtube.com/watch?v=<id>&t=<time>`), otherwise something like `/watch?v=<id>&t=<time>` won't make sense to the user.

Reply:
In this case, to be consistent with the link target it would be better to display something like `<currentHost>/watch?v=<id>`

Reply:
I'd prefer if we don't chznge the current behavior, which is displaying the youtube URL (like in the original description), but redirecting to the watch page.